grafana/containers/goldensignals4.json

```diff
@@ -0,0 +1,531 @@
+{
```

Comment:
We need better name for this dashboard. Golden for X may not be Golden for Y.

Reply:
Site-reliability engineering and others are using this name as a standard. It gives you a quick view of the health of a system.
We do not have metrics to show the latency histograms. But, this is the best we can show today

Reply:
Which Site-reliability engineering ?

Reply:
Check out this link for more details. https://landing.google.com/sre/sre-book/chapters/monitoring-distributed-systems/

Reply:
"Database Reliability" might be better name per my $0.02